Content.Server/GameTicking/GameRules/RuleTraitor.cs

```diff
@@ -0,0 +1,119 @@
+﻿using System;
+using System.Threading;
+using Content.Server.Interfaces.Chat;
+using Content.Server.Interfaces.GameTicking;
+using Content.Server.Mobs.Roles.Traitor;
+using Content.Server.Players;
+using Content.Shared;
+using Content.Shared.GameObjects.Components.Damage;
+using Robust.Server.GameObjects.EntitySystems;
+using Robust.Server.Interfaces.Player;
+using Robust.Shared.Audio;
+using Robust.Shared.GameObjects.Systems;
+using Robust.Shared.Interfaces.Configuration;
+using Robust.Shared.IoC;
+using Robust.Shared.Localization;
+using Timer = Robust.Shared.Timers.Timer;
+
+namespace Content.Server.GameTicking.GameRules
+{
+    public class RuleTraitor : GameRule
+    {
+        private static readonly TimeSpan DeadCheckDelay = TimeSpan.FromSeconds(1);
+
+        [Dependency] private readonly IChatManager _chatManager = default!;
+        [Dependency] private readonly IPlayerManager _playerManager = default!;
+        [Dependency] private readonly IGameTicker _gameTicker = default!;
+        [Dependency] private readonly IConfigurationManager _cfg = default!;
+
+        private readonly CancellationTokenSource _checkTimerCancel = new CancellationTokenSource();
+
+        public override void Added()
+        {
+            _chatManager.DispatchServerAnnouncement(Loc.GetString("Hello spessmen! Have a good shift!"));
+
+            bool Predicate(IPlayerSession session) => session.ContentData()?.Mind?.HasRole<TraitorRole>() ?? false;
+
+            EntitySystem.Get<AudioSystem>().PlayGlobal("/Audio/Misc/tatoralert.ogg", AudioParams.Default, Predicate);
+
+            //not sure if i should hook this into the healthchange entityevent or do it this way?
+            Timer.SpawnRepeating(DeadCheckDelay, _checkWinConditions, _checkTimerCancel.Token);
+        }
+
+        public override void Removed()
+        {
+            base.Removed();
+
+            _checkTimerCancel.Cancel();
+        }
+
+        private void _checkWinConditions()
```

Comment:
I would personally not have a check win condition for when traitors are dead. If we want to encourage people to hunt the traitors and do nothing else, there's SSS mode